.github/workflows/storybook.yml

```
@@ -0,0 +1,22 @@
+
+permissions: # so we can upload the storybook
+  contents: write
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                node-version: lts/*
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)